testmodule/fileparser.py

```diff
@@ -0,0 +1,87 @@
+"""fileparser.py contains a class to test if given .JBI file is operable."""
+class jobFile:
```

Comment:
```suggestion
class JobFile:
```

The standard Python naming convention is to use *CapWords* style for classes, in this case `JobFile`

see https://peps.python.org/pep-0008/#class-names